AsyncDisplayKit/ASTableView.h

```diff
@@ -272,6 +272,17 @@
 @optional
 
 /**
+ * Provides the constrained size range for measuring the node at the index path.
+ *
+ * @param tableView The sender.
```

Comment:
Delete these two @param lines and the extra whitespace, so there is just a single empty line between the "Provides ..." and "@returns ..." line.  These particular comments don't add value and make an already large header even bigger.